### ISSUE TITLE
cannot install connect using npm

### ISSUE THREAD

**Issue by MPE2016** [senchalabs/connect#1110]
this is the error readout. I see, of course, that it has something to do with permissions and the path, but I can't really say what is causing it. 

Help is much appreciated!

npm WARN checkPermissions Missing write access to /home/maarten/node_modules/connect
npm WARN enoent ENOENT: no such file or directory, open '/home/maarten/package.json'
npm WARN maarten No description
npm WARN maarten No repository field.
npm WARN maarten No README data
npm WARN maarten No license field.

npm ERR! path /home/maarten/node_modules/connect
npm ERR! code ELOOP
npm ERR! errno -40
npm ERR! syscall access
npm ERR! ELOOP: too many symbolic links encountered, access '/home/maarten/node_modules/connect'

**Comment by dougwilson** [comment]
Hi! I'm sorry you're having trouble. I tried to install and didn't have an issue. Are you able to install a different module?

**Comment by MPE2016** [comment]
Hi!,

okay, that's definitely one piece of the puzzle. I can't seem to install
any module at all.

So I guess my install of npm is probably the culprit. Any thoughts on
that?

Thanks a million!

M



On Wed, 24 Oct 2018 05:04:42 -0700, Douglas Wilson wrote:

> Hi! I'm sorry you're having trouble. I tried to install and didn't have
an issue. Are you able to install a different module?

**Comment by MPE2016** [comment]
Hi Douglas, I think I've fixed it sorry for wasting your time,

Well I guess this proves that I'm still a complete moron when it comes to
nodejs. I just realised that I need to place a new nodejs project in its
own folder (i.e. not the home directory). Then, after running the 'npm
init' command I was able to install connect just fine.

Thanks again for responding, and once again sorry for wasting your time.

All the best,

MPE



On Wed, 24 Oct 2018 05:04:42 -0700, Douglas Wilson wrote:

> Hi! I'm sorry you're having trouble. I tried to install and didn't have
an issue. Are you able to install a different module?
